samples/tooling/profiles/profiles.cpp

```diff
@@ -98,9 +96,9 @@ std::unique_ptr<vkb::Device> Profiles::create_device(vkb::PhysicalDevice &gpu)
 
```

Comment:
Maybe move the profile support check to the beginning of this function. None of the stuff above this check is needed for it.

Reply:
That's a goo point. Adjusted the order.